File: sdk/scalaz/src/main/scala/org/openapitools/client/api/InlineResponse200.scala
package org.openapitools.client.api

import argonaut._
import argonaut.EncodeJson._
import argonaut.DecodeJson._

import org.http4s.{EntityDecoder, EntityEncoder}
import org.http4s.argonaut._
import org.joda.time.DateTime
import InlineResponse200._

case class InlineResponse200 (
  /* A list of UPCs */
  upcs: Option[List[String]],
/* A list of EANs */
  eans: Option[List[String]],
/* A list of ASINs */
  asins: Option[List[String]],
message: Option[String])

object InlineResponse200 {
  import DateTimeCodecs._

  implicit val InlineResponse200CodecJson: CodecJson[InlineResponse200] = CodecJson.derive[InlineResponse200]
  implicit val InlineResponse200Decoder: EntityDecoder[InlineResponse200] = jsonOf[InlineResponse200]
  implicit val InlineResponse200Encoder: EntityEncoder[InlineResponse200] = jsonEncoderOf[InlineResponse200]
}

File: sdk/javascript-closure-angular/API/Client/InlineResponse200.js
goog.provide('API.Client.inline_response_200');

/**
 * @record
 */
API.Client.InlineResponse200 = function() {}

/**
 * A list of UPCs
 * @type {!Array<!string>}
 * @export
 */
API.Client.InlineResponse200.prototype.upcs;

/**
 * A list of EANs
 * @type {!Array<!string>}
 * @export
 */
API.Client.InlineResponse200.prototype.eans;

/**
 * A list of ASINs
 * @type {!Array<!string>}
 * @export
 */
API.Client.InlineResponse200.prototype.asins;

/**
 * @type {!string}
 * @export
 */
API.Client.InlineResponse200.prototype.message;

